lib/src/extras/logging.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(unexpected_cfgs)]
```

Comment:
Could you fix this warning instead? I think the warning can be quite useful in detecting typos.

Reply:
It is regarding the `tokio_unstable` feature that isn't defined in the `Cargo.toml` but can be passed as a `RUSTFLAGS` during compilation.

Another way I could possibly fix this is by adding a workspace lint that was introduced on purpose for the `unexpected_cfgs` https://github.com/rust-lang/cargo/pull/13913/. Do you prefer that?

Reply:
Yes, I prefer that. `#[allow(unexpected_cfgs)]` does not fix the warning but just hides it. You can use the lint configuration you linked to declare what cfgs we expect in our code. This will allow us to catch typos, because it's declaring a whitelist of lints we expect.